src/migrations/env.py

```
@@ -23,24 +23,44 @@
 # for 'autogenerate' support
 # from myapp import mymodel
 
-# noinspection PyTypeChecker
+# noinspection PyUnresolvedReferences
 from scheme.chat.chat_scheme import Chat
```

Ruff

`scheme.chat.chat_scheme.Chat` imported but unused

Remove unused import: `scheme.chat.chat_scheme.Chat`

(F401)